### PR TITLE
feat: configurable server port

### PR DESCRIPTION
**Pull request by JanPetr**

### CODE REVIEW

File: src/btwa_api/web.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def __init__(self, statsd, status_handler, trace_handler, port=8080, loop=None):
 
     async def start(self):
         self.web_app_handler = self.web_app.make_handler()
-        self.logger.info("Starting webserver")
+        self.logger.info("Webserver started on port %s", self.port)
```
Comment (jendakol) [inline src/btwa_api/web.py:33]: Please try to detect binding failure too.

**Review by jendakol — CHANGES_REQUESTED** [review]

File: src/btwa_api/web.py (hunk 1 above)
Comment (JanPetr) [reply src/btwa_api/web.py:33]: I do, the PR is coming. I'll stack in on top of this one. Logically it's not the same :))

Comment (JanPetr) [reply src/btwa_api/web.py:33]: https://github.com/covid19cz/bt-tracing-webapp/pull/14

**Comment by jendakol** [comment]
Closing as it's obsolete.

**Pull request closed without merging**